src/foam/nanos/script/Script.js

```diff
@@ -273,11 +273,11 @@ foam.CLASS({
           this.status = this.ScriptStatus.RUNNING;
           this.runScript().then(() => {
             this.status = this.ScriptStatus.UNSCHEDULED;
-            this.scriptDAO.put(this);
+            this.scriptDAO.put(this.fclone());
```

Comment:
How did we update the status on the previous line if 'this' was frozen?

Reply:
another entry from testing.  needs to be undone.

Reply:
I've reverted this, but I don't know how to fix this, as copyFrom should always fail, unless 'this.frozen' is false.